modules/serverless-install-eventing-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-eventing.adoc
+
+[id="serverless-install-eventing-web-console_{context}"]
+= Installing Knative Eventing using the web console
+
+.Procedure
+. Navigate in the web console to the *Catalog* → *OperatorHub* → *Installed Operators* page.
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
+. Click *Knative Eventing* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Eventing* tab.
++
+image::serverless-installed-operator.png[Installed Operators page]
+. Click the *Create Knative Eventing* button.
++
+image::serverless-create-eventing.png[Knative Eventing tab]
+. In the *Create Knative Eventing* page, you can choose to configure the `KnativeEventing` object by using either the default form provided, or by editing the YAML.
++
+* Using the form is recommended for simpler configurations that do not require full control of `KnativeEventing` object creation.
+.. Optional. If you are configuring the `KnativeEventing` object using the form, make any changes that you want to implement for your Knative Eventing deployment.
```

Comment:
This shows up nested one level lower. Add a "+" to the line above to match what you have here: https://github.com/openshift/openshift-docs/pull/21074/files#diff-891c11a894f1badc4c64e2ed65664b85R21.